server/src/test/java/org/candlepin/dto/api/v1/ProductContentTranslatorTest.java

```diff
@@ -0,0 +1,109 @@
+        modelTranslator.registerTranslator(new ContentTranslator(), Content.class, ContentDTO.class);
+        modelTranslator.registerTranslator(this.translator, ProductContent.class, ProductContentDTO.class);
+    }
+
+    @Override
+    protected ProductContentTranslator initObjectTranslator() {
+        return this.translator;
+    }
+
+    @Override
+    protected ProductContent initSourceObject() {
+        Content content = new Content();
+
+        content.setUuid("test_value");
+        content.setId("test_value");
+        content.setType("test_value");
+        content.setLabel("test_value");
+        content.setName("test_value");
+        content.setVendor("test_value");
+        content.setContentUrl("test_value");
+        content.setRequiredTags("test_value");
+        content.setReleaseVersion("test_value");
+        content.setGpgUrl("test_value");
+        content.setMetadataExpiration(1234L);
+        content.setModifiedProductIds(Arrays.asList("1", "2", "3"));
+        content.setArches("test_value");
+        content.setLocked(Boolean.TRUE);
+
+        ProductContent source = new ProductContent();
+        source.setContent(content);
+        source.setEnabled(true);
+
+        return source;
+    }
+
+    @Override
+    protected ProductContentDTO initDestinationObject() {
+        // Nothing fancy to do here.
+        return new ProductContentDTO();
+    }
+
+    @Override
+    protected void verifyOutput(ProductContent source, ProductContentDTO dto, boolean childrenGenerated) {
+        if (source != null) {
+            // This DTO does not have any nested objects, so we don't need to worry about the
+            // childrenGenerated flag
+
+            assertEquals(source.isEnabled(), dto.getEnabled());
+
+            if (childrenGenerated) {
+                Content content = source.getContent();
+                ContentDTO contentDto = dto.getContent();
+                assertEquals(content.getUuid(), contentDto.getUuid());
+                assertEquals(content.getId(), contentDto.getId());
+                assertEquals(content.getType(), contentDto.getType());
+                assertEquals(content.getLabel(), contentDto.getLabel());
+                assertEquals(content.getName(), contentDto.getName());
+                assertEquals(content.getVendor(), contentDto.getVendor());
+                assertEquals(content.getContentUrl(), contentDto.getContentUrl());
+                assertEquals(content.getRequiredTags(), contentDto.getRequiredTags());
+                assertEquals(content.getReleaseVersion(), contentDto.getReleaseVer());
+                assertEquals(content.getGpgUrl(), contentDto.getGpgUrl());
+                assertEquals(content.getMetadataExpiration(), contentDto.getMetadataExpire());
+                assertEquals(content.getModifiedProductIds(), contentDto.getModifiedProductIds());
+                assertEquals(content.getArches(), contentDto.getArches());
```

Comment:
Code duplication: This can be replaced by calling `ContentTranslatorTest`'s `verifyOutput` method

Reply:
Done.